crates/opencascade-sys/src/lib.rs

```diff
@@ -32,6 +32,14 @@ pub mod ffi {
         IFSelect_RetStop,
     }
 
+    #[derive(Debug)]
+    #[repr(u32)]
+    pub enum BOPAlgo_GlueEnum {
```

Comment:
I found a good [article](https://dev.opencascade.org/content/gluing-options-boolean-component) talking about this a bit.